### DISCUSSION TITLE
The cursor remains on TeXstudio after the Zotero picker window shows up

### DISCUSSION THREAD

**Discussion by RabahHouna** [retorquere/zotero-better-bibtex#2535] [Q&A] [answered]
Hello, I have a problem. I used this https://forums.zotero.org/discussion/comment/389917/#Comment_389917 to show up the Zotero picker window in front of TeXstudio. But when the Zotero picker window shows up, the cursor remains on TeXstudio. Consequently, I have to manually navigate to the Zotero picker using the mouse and click on it. Is there a solution to this issue?

**Comment by retorquere** [comment]
I'll have to ask the Zotero developers

**Reply by RabahHouna** [reply, marked as answer]
Thank you! I've found a solution by creating a Python file with the help of ChatGPT. The file brings the 'Quick Format Citation' window to the front based on its title. I incorporated this script into Texstudio and added a delay function to ensure proper execution. Additionally, I use an openApp.pyw script to open Zotero if it's closed. I hope this solution proves helpful to others.
